ansible/vars/topo_wan-4link.yml

```diff
@@ -0,0 +1,102 @@
+topology:
+  topo_type: wan
+  VMs:
+    ARISTA01T1:
+      vlans:
+        - 0
+        - 1
+        - 2
+        - 3
+      vm_offset: 0
+    ARISTA02T1:
+      vlans:
+        - 10
+        - 11
+        - 12
+        - 13
+      vm_offset: 1
+  DUT:
+    loopback:
+      ipv4:
+        - 10.1.0.32/32
+      ipv6:
+        - FC00:1::32/128
+wan_dut_configuration:
+  vlab-01:
```

Comment:
I don't think it is a good idea to have hostname `vlab-01` hardcoded in topo definition file. Need to use something more generic.

Reply:
use a more generic name: `WANDUT01` and add `dut_offset` to represent the index in inventory file.